public/fonts/luminara.font.css

```diff
@@ -0,0 +1,28 @@
+@import url('https://fonts.googleapis.com/css2?family=Arsenal:ital,wght@0,400;0,700;1,400;1,700&family=Libre+Caslon+Display&display=swap');
+
+@font-face {
+  font-family: 'Custom';
```

Comment:
This is because how our "Custom" font family works. We can remove this once we add proper fonts support, see https://github.com/metabase/metabase/issues/44394